src/service/document-info-impl.js

```diff
@@ -181,3 +191,20 @@ function getMetaTags(doc) {
   }
   return metaTags;
 }
+
+/**
+ * Attempts to retrieve extra parameters from the "amp_r" query param,
+ * returning an empty result if invalid.
+ * @param {!./ampdoc-impl.AmpDoc} ampdoc
+ * @return {!JsonObject<string, string|!Array<string>>}
+ */
+function getExtraParams(ampdoc) {
```

Comment:
`getReplaceParams()`

Let's refer to `amp_r` consistently as "replace params" instead as sometimes "extra" and sometimes "replace".

Reply:
Replaced everywhere